modules/shown_section.js

```diff
@@ -1,32 +1,15 @@
-import {
-  booksSection, addBookSection, contactSection, listLink, addNewLink, contactLink,
-} from './varibles.js';
-
-// Single Page Application
-export default function showSection() {
-  if (window.location.hash === '#books-header') {
-    booksSection.style.display = 'block';
-    addBookSection.style.display = 'none';
-    contactSection.style.display = 'none';
-
-    listLink.style.fontWeight = 'bold';
-    addNewLink.style.fontWeight = 'normal';
-    contactLink.style.fontWeight = 'normal';
-  } else if (window.location.hash === '#form') {
-    addBookSection.style.display = 'block';
-    booksSection.style.display = 'none';
-    contactSection.style.display = 'none';
-
-    listLink.style.fontWeight = 'normal';
-    addNewLink.style.fontWeight = 'bold';
-    contactLink.style.fontWeight = 'normal';
-  } else if (window.location.hash === '#contact') {
-    contactSection.style.display = 'block';
-    booksSection.style.display = 'none';
-    addBookSection.style.display = 'none';
-
-    listLink.style.fontWeight = 'normal';
-    addNewLink.style.fontWeight = 'normal';
-    contactLink.style.fontWeight = 'bold';
-  }
+// Show a Section after clicking a link and highlight the link
+export default function showSection(
```

Comment:
- [ ] It would be better to use arrow functions here too, as this is a better practice in ES6, please do this for all the functions.